checks/collector.py

```diff
@@ -368,7 +371,19 @@ def run(self, checksd=None, start_event=True):
             payload['metrics'].extend(self._agent_metrics.check(payload, self.agentConfig,
                 collect_duration, self.emit_duration))
 
-
+        # If required by the user, let's create the app:xxx host tags
+        if self.agentConfig['create_app_tags']:
+            running_apps = [APPTAG.format(c.name) for c in self.initialized_checks_d]
+            running_apps.extend([APPTAG.format(cname) for cname in self._get_jmx_appnames()])
+            
+            if 'host-tags' not in payload:
+                payload['host-tags'] = dict()
+            if 'system' not in payload['host-tags']:
+                payload['host-tags']['system'] = list()
+            
+            payload['host-tags']['system'].extend(running_apps)
```

Comment:
so you can simply rewrite this block with
```python
payload['host-tags']['system'] = running_apps
```

Also, running_apps is not the best name for this list i think.

Something like:
```
app_tags_list
``` 
would be more explicit